_docs/concepts/what-is-istio/comparison.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio vs. Other Software
+overview: A comparison between Istio's and similar software.
+              
+order: 30
+
+layout: docs
+type: markdown
+---
+
+This page provides a comparison between Istio and similar software.
+
+## Istio vs. Envoy
+
+Envoy is one of the building blocks of Istio. In Istio, an Envoy instance is deployed as a sidecar to every running service. Together, this set of Envoy instances makes up the **data plane**.
+
```

Comment:
Istio ingress also uses envoy

Reply:
It is helpful to mention the drawbacks using Envoy by itself without the control plane which goes to why you need the control plane.